solr/solr-ref-guide/modules/upgrade-notes/pages/major-changes-in-solr-10.adoc

```diff
@@ -111,6 +111,8 @@ Users who that don't need to vary JAR access on a per-core basis have several op
 
 * CurrencyField has been removed.  Users should migrate to the `CurrencyFieldType` implementation.
 
+* The `addHttpRequestToContext` option in `solrconfig.xml` has been removed; it's obsolete.
```

Comment:
[0] Might be worth augmenting these docs so that users know they can still get at the request object via `SolrQueryRequest.getHttpSolrCall().getReq()`